provision/testdata/base_case.kube.yaml

```diff
@@ -1359,7 +1359,7 @@ data:
                     "external_networks": [
                         "default"
                     ]
-                }
+                },
```

Comment:
The comma here will make the JSON invalid

Reply:
make sense as l3out is mandatory field handling the last item without comma